changelog.txt

```diff
@@ -1,8 +1,20 @@
+Version 2.0.4
```

Comment:
@rpdome I see this got merged but I believe this should be `2.0.5` right?

Reply:
Yup. I fixed that separately.